include/inviwo/core/datastructures/camera.h

```diff
@@ -73,13 +73,13 @@ class IVW_CORE_API Camera : public Serializable {
     virtual void configureProperties(CompositeProperty* comp, Config config) = 0;
 
     const vec3& getLookFrom() const;
-    void setLookFrom(vec3 val);
+    virtual void setLookFrom(vec3 val);
```

Comment:
Do these need to be virtual?

Reply:
Yes! Different cameras do different things. In particular perspective camera vs. orthographic. That was one of the reasons why panning in orthographic was broken.